### ISSUE TITLE
Keras Training Plot history issue: kernal die on Mac Anaconda environment when plot

### ISSUE THREAD

**Issue by Garfield-kh** [keras-team/keras#11815]
I am new to Keras, Sorry if I put on wrong place about this question.
I try Keras tutorial [https://www.tensorflow.org/tutorials/keras/basic_regression](url) on Mac Pro Anaconda Spyder and Jupyter.
But I get a problem that after model.fit, when trying cell:

`import matplotlib.pyplot as plt`

`def plot_history(history):`
  `plt.figure()`
` plt.xlabel('Epoch')`
 `plt.ylabel('Mean Abs Error [1000$]')`
 `plt.plot(history.epoch, np.array(history.history['mean_absolute_error']),
           label='Train Loss')`
`  plt.plot(history.epoch, np.array(history.history['val_mean_absolute_error']),
           label = 'Val loss')`
 ` plt.legend()`
 ` plt.ylim([0, 5])`

`plot_history(history)`

The kernal will die, if I show the content of `history.history['mean_absolute_error'],` the content inside is fine. and I can even plot it if I copy the content manually to a variable `a` and replace the `history.history['mean_absolute_error']` with `a`.

If I try virtual environment Jupyter Notebook on Mac, or anaconda on Ubuntu 16.04; the tutorial works okay.

Does anyone meet same issue as me?

Thanks for help.

**Comment by Garfield-kh** [comment]
Thank you for your reply.
I tried tensorflow code example without keras. It still died.
Re-install the whole mac os still not work.

Not sure where get problem. I go and try asking in matplotlib.

Thank you so much!

**Comment by gabrieldemarmiesse** [comment]
No problem, just a reminder, issues are for bugs in the keras code base. When in doubt, ask Stackoverflow :)
